apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/PromptResourceTest.java

```diff
@@ -93,7 +93,7 @@ class PromptResourceTest {
     private static final TestDropwizardAppExtension app;
 
     private static final WireMockUtils.WireMockRuntime wireMock;
-    private static final String[] IGNORED_FIELDS = {"latestVersion", "template"};
+    private static final String[] IGNORED_FIELDS = {"latestVersion", "template", "metadata", "changeDescription"};
```

Comment:
In this case these fields are present in `Prompt` object as `Write` only, since they are actually used for creation of `PromptVersion`. And when prompts are returned by GET requests these fields are always null, so excluded from `Prompt` assertions. They are asserted for `PromptVersion` objects.